src/Crawler.php

```diff
@@ -74,18 +74,24 @@ public function crawlSite( string $static_site_path ) : void {
         foreach ( CrawlQueue::getCrawlablePaths() as $root_relative_path ) {
             $absolute_uri = new URL( $site_path . $root_relative_path );
 
+            $crawled_contents = $this->crawlURL( $absolute_uri );
+
+            if ( !is_null($crawled_contents)) {
+                $page_hash = md5( $crawled_contents );
+            } else {
+                $page_hash = md5('');
```

Comment:
Might we hardcode this to `d41d8cd98f00b204e9800998ecf8427e` to save an operation?

Reply:
sure could